contrib/README.md

```diff
@@ -0,0 +1,84 @@
+# Contrib
+
+The goal of this document is to explain the procedure to add custom features to the spoke clusters.
+
+- [Contrib](#contrib)
+- [1. Introduction to contrib template](#1-introduction-to-contrib-template)
+- [2. Create contrib feature](#2-create-contrib-feature)
+- [3. Create pipeline tasks](#3-create-pipeline-tasks)
+- [4. Add tasks to the spoke pipeline](#4-add-tasks-to-the-spoke-pipeline)
+
+
+# 1. Introduction to contrib template
+
+The specs of a contrib feature is as desribed in the template, this is a basic structure to use as example to implement custom features to be added in the spokes clusters.
+
+```
+template
+├── deploy.sh
+├── manifests
+│   └── 01-template-namespace.yaml
```

Comment:
add a a line with" ..." after 01-template-namespace.yaml to give them context if they want to add more yaml files

Reply:
but the thing is that the template has only those files, and in this section are described the content of the template. What can be done is to add in the second step a tree capture of the new feature to show that is posible to add more manifests